dsl.go

```diff
@@ -771,7 +775,7 @@ func init() {
 
 			firstParsed, parseErr := version.NewVersion(toString(args[0]))
 			if parseErr != nil {
-				return nil, parseErr
+				return nil, ErrParsingArg
```

Comment:
What do you think about wrapping this error, so also the detail about the specific parsing issue would be propagated upstream and still `errors.Is(..)` and `errors.As(...)` could be used in the errors chain

Reply:
done